src/OSPSuite.Infrastructure.Export/DataRepositoryExportTask.cs

```diff
@@ -108,8 +108,15 @@ private static string tableNameFrom(DataColumn baseGrid)
          var tableName = baseGrid.Repository != null ? baseGrid.Repository.Name : defaultTableName;
          if (string.IsNullOrEmpty(tableName))
             tableName = defaultTableName;
+         var correctedTableName = new string(FileHelper.RemoveIllegalCharactersFrom(tableName).Take(Constants.MAX_NUMBER_OF_CHAR_IN_TABLE_NAME).ToArray());
 
-         return new string(FileHelper.RemoveIllegalCharactersFrom(tableName).Take(Constants.MAX_NUMBER_OF_CHAR_IN_TABLE_NAME).ToArray());
+         while (correctedTableName.EndsWith("'") && correctedTableName.Length > 0)
```

Comment:
if `correctedTableName.EndsWith("'")` is **true** then `correctedTableName.Length` cannot be 0, thus the 2nd condition can be removed, right?

Reply:
true. corrected